azure-mgmt-cosmosdb/src/main/java/com/microsoft/azure/management/cosmosdb/ExtendedResourceProperties.java

```diff
@@ -18,80 +18,80 @@
     /**
      * A system generated property. A unique identifier.
      */
-    @JsonProperty(value = "rid")
-    private String rid;
+    @JsonProperty(value = "_rid")
```

Comment:
@anuchandy can you look at what caused this change?

Reply:
As I checked the cosmos-db.json, these properties are always start with `-`, not sure how the value is `rid` in our code for json property.

Which way should we honor here?

Reply:
The swagger indeed has underscore as part of property names https://github.com/Azure/azure-rest-api-specs/blob/99d1cd23e2058369ea1d0bba1ad042f69503a18e/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json#L3935